Languages/german.lang

```diff
@@ -321,7 +321,7 @@ msgid "Block Updates"
 msgstr "Blocke Updates"
 
 msgid "Boot Content"
-msgstr "Boote Inhalt"
+msgstr "Boot-Inhalt"
```

Comment:
`Boot` is an action.

Reply:
I changed it to your suggestion a few weeks ago already but it does not show up in "Files changed".